snap/snapcraft.yaml

```diff
@@ -44,11 +70,25 @@ parts:
     plugin: nil
     stage-packages:
       - python3-gi
-      - python3-cairo
+      - python3-gi-cairo
+    override-stage: |
+      craftctl default
```

Comment:
I think with this change we might be able to turn on UEFI capsule splash in the build arguments, right?